profile/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Thanik Bhongbhibhat <email@thanik.me>",
   "license": "MIT",
   "scripts": {
-    "start": "parcel serve './src/index.html' --port 8000 --lazy"
+    "start": "parcel serve './src/index.html' --public-url \"/service/profile\" --port 8000 --lazy"
```

Comment:
Hack by hardcoding the proxied path for now. Profile shouldn't know this but I have to figure out static path routing in HTML in reverse proxy setup better.